includes/pages/user_shifts.php

```diff
@@ -43,18 +43,21 @@ function update_ShiftsFilter_timerange(ShiftsFilter $shiftsFilter, $days) {
   $start_time = $shiftsFilter->getStartTime();
   if ($start_time == null) {
     $start_time = time();
+    if($start_time < 1495238400) {
```

Comment:
Yeah, I was in a train without internet connection so this was the dirty but working solution. This is a dirty fix, anyway, so I think this is OK enough.